services/horizon/internal/ingest/main_test.go

```diff
@@ -55,12 +55,12 @@ func TestTick(t *testing.T) {
 	sys := sys(tt, false)
 
 	// ingest by tick
-	s := sys.Tick()
+	s := sys.Tick("GBD53ST2UUC5VESGEFIPSNZPSVIHIVV5G3SFCCMGNWD7ETMDV3HNA2JV")
```

Comment:
do we want to support a scenario where there is no white list?

Reply:
This text function expect to receive a string, and the design of our web uses whitelist.

Reply:
let me rephrase my question: what happens if we dont configure a whitelist? does horizon crash?